bundles/org.openhab.binding.smartthings/SmartthingsInstallation.md

```diff
@@ -0,0 +1,79 @@
+# Installation of Smartthings code
```

Comment:
I'm not sure if the location of the additional readme files, at the same level as the Readme.md is correct. @cpmeister Would `doc/` fit better?

Reply:
I believe that would be the correct place, yes.